app/add-details/page.tsx

```diff
@@ -0,0 +1,152 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { UUID } from 'crypto';
+import { updateUserPlants } from '@/api/supabase/queries/updateUserPlants';
+import PlantDetails from '@/components/PlantDetails';
+import { Plant, UserPlants } from '@/types/schema';
+
+export default function Home() {
+  const [currentIndex, setCurrentIndex] = useState<number>(1);
```

Comment:
Would it be possible to refactor `currentIndex` such that it starts at 0 instead of 1? I notice there's a lot of `currentIndex - 1` throughout your code, wondering if it would be easier to start at 0. If it messes with the logic somehow then it's alright

Reply:
yea i lwk changed it from 0 cuz i was being dumb and then was lazy in refactoring ill fix that